fpga/src/main/scala/arty100t/HarnessBinders.scala

```diff
@@ -41,3 +43,43 @@ class WithArty100TDDRTL extends HarnessBinder({
     ddrClientBundle <> port.io
   }
 })
+
+// Uses PMOD JA/JB
+class WithArty100TSerialTLToGPIO extends HarnessBinder({
+  case (th: HasHarnessInstantiators, port: SerialTLPort) => {
+    val artyTh = th.asInstanceOf[LazyRawModuleImp].wrapper.asInstanceOf[Arty100THarness]
+    val harnessIO = IO(port.io.cloneType).suggestName("serial_tl")
+    harnessIO <> port.io
+    val clkIO = IOPin(harnessIO.clock)
+    val packagePinsWithPackageIOs = Seq(
+      ("G13", clkIO),
+      ("B11", IOPin(harnessIO.bits.out.valid)),
+      ("A11", IOPin(harnessIO.bits.out.ready)),
+      ("D12", IOPin(harnessIO.bits.in.valid)),
+      ("D13", IOPin(harnessIO.bits.in.ready)),
+      ("B18", IOPin(harnessIO.bits.out.bits, 0)),
+      ("A18", IOPin(harnessIO.bits.out.bits, 1)),
+      ("K16", IOPin(harnessIO.bits.out.bits, 2)),
```

Comment:
Might be nice to make the TL width parametrizable to some degree, though not sure if there's a good way to do that.

Reply:
I don't think there's a good way to do this, since there's a lmited set of GPIOs, and the GPIOs might be used for other purposes.